meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in

```diff
@@ -196,13 +200,15 @@ if [ -z "${mmcPath}" ]; then
 	error_exit "Can't detect UMS exposed device!"
 fi
 
-read -p "Confirm flashing ${wicFile} to ${mmcPath} (y/N) >" -n 1 -r
+if [[ $nonInteractive != "y" ]]; then
+    read -p "Confirm flashing ${wicFile} to ${mmcPath} (y/N) >" -n 1 -r
+fi
 echo ""
-if [[ $REPLY =~ ^[Yy]$ ]]; then
+if [[ $REPLY =~ ^[Yy]$ ]] || [[ $nonInteractive = "y" ]]; then
```

Comment:
IMO vice-versa better:
`if [[ $nonInteractive = "y" ]] || [[ $REPLY =~ ^[Yy]$ ]];` 

as we can just skip checking `$REPLY` if `$nonInteractive = "y"` is true

Reply:
Fixed.

Reply:
hm, strange, I still see the old change

Reply:
My mistake. I added it in one commit and reverted in the next one. Fixed now.